helpers/config/config_test.go

```diff
@@ -88,6 +88,7 @@ type testConfig struct {
 	IncludeVolumeServices           *bool `json:"include_volume_services,omitempty"`
 	IncludeZipkin                   *bool `json:"include_zipkin,omitempty"`
 	IncludeWindows                  *bool `json:"include_windows,omitempty"`
+	IncludeIpv6         			*bool `json:"include_ipv6,omitempty"`
```

Comment:
The indentation is off because tabs are used and above spaces are used. This can be avoided by using `go fmt`.